examples/charts/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Mesa Charts Example
+
+## Summary
+
+A modified version of the "bank_reserves" example made to provide examples of mesa's charting tools.
```

Comment:
Are there any modifications to the bank_reserves? If not, why not include the charts there? Also I agree that "charts example" would be more visible (maybe remove bank_reserves?)

Reply:
My initial decision to keep the charts out of "bank_reserves" was just to avoid the bloating of bank_reserves. If we are assuming that the example is to show how mesa could be used "in the real world", it makes no sense to have four different charts, three of which tell you the exact same thing. I created the charts example to explicitly show the four kinds of charts that mesa has available: (copy pasted from the charts documentation):

- Line Charts for time-series data of multiple model parameters
- Pie Charts for model parameters
- Bar charts for both model and agent-level parameters

In my opinion, it is better to keep charts separate (as I just used "bank_reserves" because it already existed.) 

As to removing bank reserves, that would be up to y'all as project organizers, but if it served a purpose before it stands to reason that it would continue serving a purpose. (I certainly never felt there were too many examples when i was using NetLogo in my ecology class.)